ui/src/main.rs

```diff
@@ -484,8 +484,20 @@ fn app(cx: Scope) -> Element {
             .map(|theme| theme.styles.clone())
             .unwrap_or_default();
 
+        let accent_color = state.read().ui.accent_color;
+        let accent_color = if let Some(color) = accent_color {
```

Comment:
This could be simplified by doing something like the following but isnt a blocker

```rust
let accent_color = accent_color.map(|color| format!(":root {{ --primary: rgb({},{},{}); }}", color.0, color.1, color.2, )).unwrap_or_default();
```

Reply:
Hopefully we'll be replacing that section of code when the todo item in the file is done so if it's not a blocker for now i'll merge it in since it'll be replaced anyhow.